subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1704,6 +1704,27 @@ static void vs_write_bd_addr(struct net_buf *buf, struct net_buf **evt)
 	ccst->status = 0x00;
 }
 
+static void vs_read_build_info(struct net_buf *buf, struct net_buf **evt)
+{
+	struct bt_hci_rp_vs_read_build_info *rp;
+
+#define BUILD_TIMESTAMP " BUILD: " __DATE__ " " __TIME__
+#define HCI_VS_BUILD_INFO "Zephyr OS v" \
+	KERNEL_VERSION_STRING BUILD_TIMESTAMP \
+	" (" CONFIG_BT_CTLR_HCI_VS_BUILD_INFO ")"
```

Comment:
No a huge fan that you get "Zephyr OS v9.x.x BUILD: foo bar ()" if the string is not configured. If the BUILD_INFO config entry is "", then we might want to skip also the " ()". Any chance we can make this conditional.

Reply:
Yep, will do that